File: app/src/main/java/org/ole/planet/myplanet/model/RealmTeamNotification.java
package org.ole.planet.myplanet.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class RealmTeamNotification extends RealmObject {
    @PrimaryKey
    String id;
    String type;
    String parentId;
    int lastCount;

    public String getId() {
        return id;
    }

    public String getParentId() {
        return parentId;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public int getLastCount() {
        return lastCount;
    }

    public void setLastCount(int lastCount) {
        this.lastCount = lastCount;
    }

    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/main/java/org/ole/planet/myplanet/model/RealmTeamNotification.kt
package org.ole.planet.myplanet.model

import androidx.room.Entity
import androidx.room.Index
import androidx.room.PrimaryKey

/**
 * Room replacement for the former Realm `RealmTeamNotification` model. Local-only (not synced or
 * uploaded); persistence goes through
 * [org.ole.planet.myplanet.data.room.dao.TeamNotificationDao].
 */
@Entity(tableName = "team_notification", indices = [Index("type")])
open class RealmTeamNotification {
    @PrimaryKey
    var id: String = ""
    var type: String? = null
    var parentId: String? = null
    var lastCount: Int = 0
}
